tests/unit/admin/views/test_verdicts.py

```diff
@@ -36,10 +38,16 @@ def test_none(self, db_request):
 
     def test_some(self, db_request):
         check = MalwareCheckFactory.create()
-        verdicts = [MalwareVerdictFactory.create(check=check) for _ in range(10)]
+        verdicts = []
+        with freeze_time(datetime.datetime.utcnow()) as frozen_time:
+            for _ in range(10):
+                verdicts.append(
+                    MalwareVerdictFactory.create(check=check, run_date=frozen_time())
```

Comment:
Do we need `freezegun` if we're just manually specifying the `run_date`?

Reply:
yes, since run_date is a server side default (within the db) freezing time doesn't actually impact it.